cli2telegram/daemon.py

```diff
@@ -0,0 +1,132 @@
+#   cli2telegram
+#  Copyright (c) 2019.  Markus Ressel
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Affero General Public License as
+#  published by the Free Software Foundation, either version 3 of the
+#  License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU Affero General Public License for more details.
+#
+#  You should have received a copy of the GNU Affero General Public License
+#  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import logging
+import sys
+import time
+import sys
+import os
+from signal import signal, SIGINT, SIGTERM
+from datetime import datetime
+
+import threading, queue
+
+from telegram.ext import Updater
+
+from cli2telegram.config import Config
+from cli2telegram.util import send_message, prepare_code_message
+
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.DEBUG)
+handler = logging.StreamHandler(sys.stdout)
+handler.setLevel(logging.DEBUG)
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+handler.setFormatter(formatter)
+LOGGER.addHandler(handler)
+
+class Daemon:
+    def __init__(self, CONFIG:Config):
```

Comment:
Not sure why you used CAPS here, should be snake case. The same for the other properties like `RUNNING` and `PIPE_NAME`